lib/components/gene/__tests__/about-test.js

```diff
@@ -0,0 +1,37 @@
+import 'react-native'
+import React from 'react'
+import About from '../about'
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer'
+
+jest.mock('../../opaque_image_view.js', () => 'ImageView')
```

Comment:
this line is important, we need to provide a mock for any native Components - in this case it will be treated as a blank component with the name `ImageView` in the snapshot